.github/workflows/truffle_ruby.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
-    - name: Install dependencies
-      run: bundle install
+        bundler-cache: true
```

Comment:
This line reverts a fix in https://github.com/omniauth/omniauth/pull/1104

Reply:
It seems to pass now: https://github.com/omniauth/omniauth/actions/runs/5129258050/jobs/9226698922
So maybe no longer necessary?

Reply:
Right, I also have closed another fix for 22.3 - https://github.com/omniauth/omniauth/pull/1105.